libs/shared/flagd-core/src/lib/feature-flag.ts

```diff
@@ -0,0 +1,44 @@
+import {FlagValue} from '@openfeature/server-sdk';
+
+/**
+ * Flagd flag configuration structure mapping to schema definition.
+ */
+export interface Flag {
+  state: string,
+  defaultVariant: string,
+  variants: any,
```

Comment:
```suggestion
  variants: {[key: string]: FlagValue},
```

I think this is a precisely correct type for this field. See [FlagValue](https://github.com/open-feature/js-sdk/blob/8e873a3d2b6b2ada0ce7913a00ee1988226a7e05/packages/shared/src/evaluation/evaluation.ts#L15) from the SDK.

Reply:
yup, this works ❤️